src/vs/workbench/contrib/terminal/browser/terminalProcessExtHostProxy.ts

```diff
@@ -102,6 +102,10 @@ export class TerminalProcessExtHostProxy extends Disposable implements ITerminal
 		}
 	}
 
+	processBinary(data: string): void {
+		throw new Error('not implemented');
+	}
```

Comment:
Let's pass through here and let the actual Pseudoterminal object fail (instead of having no ops all over the place)

Reply:
There are no listeners to `onBinary` right now by design. Don't think we want to add any until or unless this is something we want to allow. 
https://github.com/microsoft/vscode/blob/b45caa2ebeea1219818669e92e5b550b0eabd41b/src/vs/workbench/contrib/terminal/browser/terminalProcessExtHostProxy.ts#L131-L134